app/dashboard/templates/addinterest.html

```diff
@@ -0,0 +1,55 @@
+{% comment %}
+    Copyright (C) 2018 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<div id="add_interest" class="white-light-bg">
+  <div class="container-fluid">
+    <div class="row closebtn">
+      <a rel="modal:close" href="javascript:void" class="close" aria-label="Close dialog">
+        <span aria-hidden="true">&times;</span>
+      </a>
+    </div>
+    <div class="row">
+      <h5 class="interest-modal-title">Great to see that you're interested in this project!</h5>
```

Comment:
All instances of copy throughout this file should be wrapped in `{% trans "<str>" %}`

For example:

`{% trans "Great to see that you're interested in this project!" %}`